File: Ejercicio12-Condicionales.cpp
/*12. Hacer un menú que considere las siguientes opciones:
Caso 1: Cubo de un número
Caso 2: Número par o impar
Caso 3: Salir
*/

#include<iostream>
#include<math.h>
using namespace std;
int main(){
    int opcion, num;
    float numero, cubo;

    cout<<"\tMenu:"<<endl;
    cout<<"1. Cubo de un numero."<<endl;
    cout<<"2. Numero par o impar"<<endl;
    cout<<"3. Salir"<<endl;
    cout<<"Elige una opcion: ";
    cin>>opcion;

    switch(opcion){
        case 1:
            cout<<"Digite un numero: ";
            cin>>numero;
            cubo = pow(numero,3);
            cout<<"El resultado es: "<<cubo<<endl; break;

        case 2:
            cout<<"Digite un numero: ";
            cin>>num;
            if (num%2 == 0){
                cout<<"El numero es PAR"<<endl;break;
            }
            else{
                cout<<"El numero es IMPAR"<<endl;break;
            }
        case 3: break;
        default: cout<<"Escogiste una opcion NO valida"<<endl;break;
    }
    return 0;
}

File: Comparativa Python/Ejercicio12-Condicionales.py
"""
12. Hacer un menú que considere las siguientes opciones:
Caso 1: Cubo de un número
Caso 2: Número par o impar
Caso 3: Salir
"""

print('\tMenú virtual')
print('1. Cubo de un número')
print('2. Número par o impar')
print('3. Salir')
opcion = int(input('Digite opción [1-3]: '))

if opcion == 1:
    num = float(input('Digite un número: '))
    resultado = pow(num, 3)
    print('El resultado es: ', str(round(resultado, 2)))
elif opcion == 2:
    num = int(input('Digite un número: '))
    if num == 0:
        print('El número es cero')
    elif num % 2 == 0:
        print('El número es par')
    else:
        print('El número es impar')
elif opcion == 3:
    print('Fin de programa')
else:
    print('Opción No valida')